components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -188,15 +222,15 @@ typedef void (^MDCActionSheetHandler)(MDCActionSheetAction *_Nonnull action);
  Action sheet actions must have a title that will be set within actionWithTitle:image:handler:
  method.
  */
-@property (nonatomic, nonnull, readonly) NSString *title;
+@property(nonatomic, nonnull, readonly) NSString *title;
```

Comment:
Since this is an `NSString`, please have it use `copy` semantics instead of `strong` (the default).

Reply:
It should also be marked `nullable` since I see the constructor being described in the docs actually accepts `nullable` parameters. 

>  \+ (nonnull instancetype)actionSheetControllerWithTitle:(nullable NSString *)title;

Reply:
Done

Reply:
I think you may be looking at the wrong initializer. The initializer for this `title` property is `(nonnull instancetype)actionWithTitle:(nonnull NSString *)title image:(nullable UIImage *)image handler:(__nullable MDCActionSheetHandler)handler;` you are referencing the initializer for `MDCActionSheetController`, this property is in `MDCActionSheetAction`.